bibtex-actions.el

```diff
@@ -443,11 +429,9 @@ TEMPLATE."
 
 (defun bibtex-actions-citation-key-at-point ()
   "Return citation keys at point as a list for `embark'."
-  (when-let ((keys (or (bibtex-actions-get-key-org-cite)
```

Comment:
If `bibtex-actions-get-key-org-cite` is removed, `bibtex-actions-dwim` will not work in org-mode buffers since `(bibtex-completion-key-at-point)` cannot find org-cite keys by default.

Another way is adding `bibtex-actions-get-key-org-cite` to `bibtex-completion-key-at-point-functions`.

Reply:
I'm not sure the best approach; I was just asking because it's taken me too long to figure this out, and it should be simple.

WDYT?

Reply:
I think it is better to put `bibtex-actions-get-key-org-cite` in `bibtex-completion-key-at-point-functions`, together with functions like `bibtex-completion-get-key-latex`. Then `bibtex-actions` can just use `bibtex-completion-key-at-point`.

Reply:
And where/how does the embark transformations happen?

Maybe I should start another branch; this still doesn't work.

Reply:
> Maybe I should start another branch; this still doesn't work.

I tried this, with the most minimal change, but can't figure this out @ilupin.

Maybe you can, and submit a separate PR?

To summarize:

- `embark-act` at point works fine with org-cite, except ...
- .. the default action does not